cli/src/complete.rs

```diff
@@ -28,19 +29,42 @@ use crate::config::default_config;
 use crate::config::LayeredConfigs;
 use crate::ui::Ui;
 
+static BOOKMARK_HELP_TMPL: &str = r#"
+"\t" ++
+if(normal_target,
+    normal_target.change_id().shortest() ++ ": " ++
```

Comment:
nit: do we need `change_id`? As you said you aren't interesting in `commit_id`, I don't think `change_id` is useful.

Reply:
Hm, yeah. I often start to memorize change ids of commits I work on a lot. So they are more interesting to me than commit ids. But it doesn't make much sense showing them in the context of bookmarks. I'll remove it.